### DISCUSSION TITLE
🐷 Jonah: An AI Discord Bot!

### DISCUSSION THREAD

**Discussion by MilesWK** [community/community#135353] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

Hello 👋! I am MilesWK and I have been working on an AI Discord bot for everyone to use... for free!

[Jonah](https://jonah-bot.glitch.me/) is an AI Discord bot that allows you to talk with the bot and get a response. If you are direct messaging the bot, you just have to talk to it to get a reply! However, if you are in a server, you can mention it (@) to get Jonah's attention!
![image](https://github.com/user-attachments/assets/a0a19ea1-3503-475c-a59f-070fab6a041a)

Jonah also has some commands: 
- `/information`: Gives information about Jonah
- `/nuke [optional: amount]`: This commands allows administrators to remove all or a certain amount of messages in a channel.
- `/pigLatin [sentence]`: Turns a sentence into pig latin!
- `/me`: Gives detailed information about you!

Check out the [GitHub Repository](https://github.com/MilesWK/Jonah) and [Jonah's Website](https://jonah-bot.glitch.me/) for more information!

**Comment by ebndev** [comment]
Hi @MilesWK, thanks for sharing with the community! 

Unfortunately, we currently do not allow self-promotion or advertising on the Community Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.

**Reply by MilesWK** [reply]
I thought this was supposed to be show and tell section! I am showing and telling the world about jonah!
![image](https://github.com/user-attachments/assets/4350ab17-3a06-489b-8a00-b53ec09a97c3)
